### PR TITLE
Feat: addon readme

### PR DESCRIPTION
**Pull request by wangyikewxgm**
Signed-off-by: wangyike <wangyike_wyk@163.com>

### Checklist

I have:
- [ ] Title of the PR starts with OAM type (e.g. `[Workload]` or `[Trait]` or `[Scope]`).
- [ ] Updated/Added any relevant [documentation] and [examples].
- [ ] Unit/E2E Tests added.

### CODE REVIEW

File: experimental/addons/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# KubeVela experimental addons
+
+This dir is KubeVela official experimental addon-registry which will contain hasn't verified stable KubeVela addon.
+
+Addon files in this dir files will be synced to alibaba-oss but will not be set in KubeVela by default.
+
+You can add this addon registry by vela cli then use these addons.
+
+```shell
+$ vela addon registry add experimental --type=git --gitUrl=https://github.com/oam-dev/catalog/ --path=experimental/addons
```
Comment (wonderflow) [inline experimental/addons/README.md:10]: I think  we should also have an OSS bucket address here?

Comment (chivalryq) [reply experimental/addons/README.md:10]: After this PR is merged https://github.com/oam-dev/kubevela/pull/2907

Comment (wangyikewxgm) [reply experimental/addons/README.md]: Cli have support addon registry oss path by https://github.com/oam-dev/kubevela/pull/2976.

File: experimental/addons/README.md [hunk 2]
```diff
@@ -1,19 +1,32 @@
...
 
-Addon files in this dir files will be synced to alibaba-oss but will not be set in KubeVela by default.
+Addon files in this dir files will be synced to alibaba-oss. Addons in this  dir will not be set in KubeVela by default.
 
-You can add this addon registry by vela cli then use these addons.
+You can add this addon registry by vela cli and use them.
 
 ```shell
-$ vela addon registry add experimental --type=git --gitUrl=https://github.com/oam-dev/catalog/ --path=experimental/addons
+$ vela addon registry add experimental --type oss --ossEndpoint=https://addons.kubevela.net --path=experimental/
```
Comment (wonderflow) [inline experimental/addons/README.md:10]: Can we omit parse the `path` automatically？ And we already decalre it as OSS type, then we shouldn't redundant the parameter as `ossEndpoint`

The Command should be something like:

```
vela addon registry add experimental --type oss --endpoint=https://addons.kubevela.net/experimental
```

File: experimental/addons/README.md [hunk 3]
```diff
@@ -1,19 +1,32 @@
...
 ```
 
 ```shell
 $ vela addon registry list      
-Name            Type    URL                                                                
-KubeVela        Oss     https://addons.kubevela.net                                        
-experimental    Git     https://github.com/oam-dev/catalog//tree/master/experimental/addons
+Name            Type    URL                                                                                                      
+experimental    Oss     https://addons.kubevela.net
```
Comment (wonderflow) [inline experimental/addons/README.md:16]:
```suggestion
experimental    OSS     https://addons.kubevela.net 
```

OSS should all be capital case

File: experimental/addons/README.md (hunk 2 above)
Comment (chivalryq) [reply experimental/addons/README.md:10]: Actually This is not a valid path: https://addons.kubevela.net/experimental, path=experimental is used like `https://addons.kubevela.net?prefix=experimental/`. So I don't argree to let user to fill in this URL.

File: addons/README.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# KubeVela addons
+
+This dir is KubeVela official addon-registry which will contain stable KubeVela addons.
+
+Addon files in this dir files will be synced to alibaba-oss. This addon registry be set in KubeVela as default addons registry.
+
+You can list all addons by vela cli. For example
+
+```shell
+vela addon list
```
Comment (wonderflow) [inline addons/README.md:10]:
```suggestion
$ vela addon list
```

File: addons/README.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+```
+
+When user enable an addon by UX or cli, addon files will be generated as a KubeVela application and apply it.
+
+You can manage the addon-registry by vela cli.
+
+```shell
+$ vela addon registry list 
+Name            Type    URL                        
+KubeVela        Oss     https://addons.kubevela.net
```
Comment (wonderflow) [inline addons/README.md:28]:
```suggestion
KubeVela        OSS     https://addons.kubevela.net
```

**Comment by wangyikewxgm** [comment]
this pr should block until this pr: https://github.com/oam-dev/kubevela/pull/2980 merged

**Review by wonderflow — APPROVED** [review]
good job thanks

**Pull request merged**